x/ccv/consumer/keeper/genesis.go

```diff
@@ -89,7 +94,10 @@ func (k Keeper) InitGenesis(ctx sdk.Context, state *consumertypes.GenesisState)
 
 		// set provider client id
 		k.SetProviderClientID(ctx, state.ProviderClientId)
+	}
 
+	if state.PreCCV {
```

Comment:
Could we add a comment specifying why a pre CCV chain should return early here

Reply:
In case of preCCV, validator set is returned on democracy staking module and therefore consumer module is not returning genesis validator set.